igvc_gazebo/nodes/control/main.cpp

```diff
@@ -76,41 +92,51 @@ int main(int argc, char **argv)
 
   auto stateSub = handle.subscribe("/joint_states", 1, jointStateCallback);
 
-  std::chrono::time_point<std::chrono::system_clock> prev, now;
-  prev = std::chrono::system_clock::now();
+  ros::Time prev, now;
+  prev = ros::Time::now();
 
-  ros::Rate rate{ 30 };
```

Comment:
Also, this rate should match the update rate in motor_controller.launch as update rates have significant effects on a the robot’s motion